test/config/integration_tests.yml

```diff
@@ -72,6 +72,14 @@ testCases:
 - name: nested-ss
   configFile: test/config/examples/mnist-nested-search-space.yml
 
+- name: classic-nas-pytorch
+  configFile: test/config/examples/classic-nas-pytorch.yml
+  # generate search space through nnictl ss_gen
+  doSsgen: True
+  ssgenCodeDir: examples/nas/classic_nas
+  # this file is automatically generated by nnictl ss_gen
+  ssFilePath: test/config/examples/nni-nas-search-space.json
+
 
```

Comment:
Maybe we can split this test cases into 2 cases, then we will do not need handle the special key 'doSsgen' etc only for this case.
1. 
```
name: nas-gen-ss
launchCommand: nnictl ss_gen --trial_command="python3 mnist.py --epochs 1" --trial_dir=examples/nas/classic_nas --file=test/config/examples/nni-nas-search-space.json
stopCommand:
experimentStatusCheck: False
```

2.
```
name: classic-nas-pytorch
configFile: test/config/examples/classic-nas-pytorch.yml
```

Reply:
good point

Reply:
by the way, are the two cases guaranteed to be tested sequentially one after the other?

Reply:
Yes, the order is guaranteed.   You can run the 2 cases on local system to check whether the case is OK:
```
python3 nni_test/nnitest/run_test.py --config config/integration_tests.yml --case classic-nas
```

Reply:
seems stopCommand cannot be the combine of two commands....

Reply:
yes, it seems the combination of two commands not working for `stopCommand` due to the `shlex.split`,  
it seems it can be fixed by replacing `proc = subprocess.run(shlex.split(launch_command))` with:
```
proc = subprocess.run(launch_command, shell=True)
```